
C:\Users\vanja>py
Python 3.12.2 (tags/v3.12.2:6abddd9, Feb  6 2024, 21:26:36) [MSC v.1937 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> a=np.array([1,2,3])
>>> a
array([1, 2, 3])
>>> type(a)
<class 'numpy.ndarray'>
>>> b=np.array([[1,2,3],[4,5,6]])
>>> b
array([[1, 2, 3],
       [4, 5, 6]])
>>> type(b)
<class 'numpy.ndarray'>
>>> a
array([1, 2, 3])
>>> b
array([[1, 2, 3],
       [4, 5, 6]])
>>> c=np.array([[[1,2,3],[4,5,6],[7,8,9]]])
>>> c
array([[[1, 2, 3],
        [4, 5, 6],
        [7, 8, 9]]])
>>> np.arange(10)
array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
>>> n.ones(10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'n' is not defined. Did you mean: 'np'?
>>> np.ones(12)
array([1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.])
>>> np.twenty(4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\__init__.py", line 347, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'twenty'
>>> np.twentys(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\__init__.py", line 347, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'twentys'
>>> np.three(3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\__init__.py", line 347, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'three'
>>> np.threes(5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\__init__.py", line 347, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'threes'
>>> np.zeros(2)
array([0., 0.])
>>> np.ones(2)
array([1., 1.])
>>> np.ones(4,5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\numeric.py", line 191, in ones
    a = empty(shape, dtype, order)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Cannot interpret '5' as a data type
>>> np.ones((4,5))
array([[1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1.]])
>>> np.zeros((2,9))
array([[0., 0., 0., 0., 0., 0., 0., 0., 0.],
       [0., 0., 0., 0., 0., 0., 0., 0., 0.]])
>>> np.ones((4,19))
array([[1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.,
        1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.,
        1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.,
        1., 1., 1.],
       [1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1., 1.,
        1., 1., 1.]])
>>> np.arange(1,2,20)
array([1])
>>> np.arange(1,5,20)
array([1])
>>> np.arange(1,5,30)
array([1])
>>> np.arange(1,30,2)
array([ 1,  3,  5,  7,  9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29])
>>> np.arange(2,20,2)
array([ 2,  4,  6,  8, 10, 12, 14, 16, 18])
>>> np.arange(2,30,2)
array([ 2,  4,  6,  8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28])
>>> np.arange(2,40,2)
array([ 2,  4,  6,  8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34,
       36, 38])
>>> np.arange(2,50,2)
array([ 2,  4,  6,  8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34,
       36, 38, 40, 42, 44, 46, 48])
>>> a
array([1, 2, 3])
>>> b
array([[1, 2, 3],
       [4, 5, 6]])
>>> c
array([[[1, 2, 3],
        [4, 5, 6],
        [7, 8, 9]]])
>>> np.arange(9)
array([0, 1, 2, 3, 4, 5, 6, 7, 8])
>>> np.ones((3,6))
array([[1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1.],
       [1., 1., 1., 1., 1., 1.]])
>>> np.ones()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: ones() missing 1 required positional argument: 'shape'
>>> np.ones(2)
array([1., 1.])
>>> np.ones(9)
array([1., 1., 1., 1., 1., 1., 1., 1., 1.])
>>> np.arange(1,2,20)
array([1])
>>> np.arange(1,6)
array([1, 2, 3, 4, 5])
>>> np.arange(1,6,30)
array([1])
>>> np.arange(1,30,2)
array([ 1,  3,  5,  7,  9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29])
>>> np.arange(1,30,3)
array([ 1,  4,  7, 10, 13, 16, 19, 22, 25, 28])
>>> np.arange(1,30,4)
array([ 1,  5,  9, 13, 17, 21, 25, 29])
>>> np.arange(1,30,5)
array([ 1,  6, 11, 16, 21, 26])
>>> np.arange(1,30,6)
array([ 1,  7, 13, 19, 25])
>>> np.arange(1,30,7)
array([ 1,  8, 15, 22, 29])
>>> np.arange(1,30,8)
array([ 1,  9, 17, 25])
>>> np.arange(1,30,9)
array([ 1, 10, 19, 28])
>>> np.arange(1,30,10)
array([ 1, 11, 21])
>>> np.arange(1,30,11)
array([ 1, 12, 23])
>>> np.arange(1,30,12)
array([ 1, 13, 25])
>>> np.arange(1,30,13)
array([ 1, 14, 27])
>>> np.arange(1,30,14)
array([ 1, 15, 29])
>>> np.arange(1,30,15)
array([ 1, 16])
>>> np.arange(1,30,16)
array([ 1, 17])
>>> np.arange(1,30,17)
array([ 1, 18])
>>> np.arange(1,30,18)
array([ 1, 19])
>>> np.arange(1,30)
array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16, 17,
       18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29])
>>> np.arange(30)
array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13, 14, 15, 16,
       17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29])
>>> np.arange(1,30,2)
array([ 1,  3,  5,  7,  9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29])
>>> np.identity(3)
array([[1., 0., 0.],
       [0., 1., 0.],
       [0., 0., 1.]])
>>> np.identity(4)
array([[1., 0., 0., 0.],
       [0., 1., 0., 0.],
       [0., 0., 1., 0.],
       [0., 0., 0., 1.]])
>>> np.identity(5)
array([[1., 0., 0., 0., 0.],
       [0., 1., 0., 0., 0.],
       [0., 0., 1., 0., 0.],
       [0., 0., 0., 1., 0.],
       [0., 0., 0., 0., 1.]])
>>> np.arange(0,2)
array([0, 1])
>>> np.arange(1,10)
array([1, 2, 3, 4, 5, 6, 7, 8, 9])
>>> np.arange(1,3,30)
array([1])
>>> np.arange(1,30,3)
array([ 1,  4,  7, 10, 13, 16, 19, 22, 25, 28])
>>> np.ones(1,30,3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\vanja\AppData\Local\Programs\Python\Python312\Lib\site-packages\numpy\core\numeric.py", line 191, in ones
    a = empty(shape, dtype, order)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Cannot interpret '30' as a data type
>>> np.zeros(9)
array([0., 0., 0., 0., 0., 0., 0., 0., 0.])
>>> np.zeros(22)
array([0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
       0., 0., 0., 0., 0.])
>>> exit()
